### DISCUSSION TITLE
Meaning of "scenariosAvoided" parameter in output

### DISCUSSION THREAD

**Discussion by metallicalfa2** [artilleryio/artillery#946] [Artillery]
I have been using artillery for load testing an API server. In test results, the "scenariosAvoided" parameter is usually high. Could someone please point me to the documentation of the parameter?

**Comment by hassy** [comment]
hi @rathishubham7, "scenarios avoided" is the count of virtual users which would've been launched but were not because `maxVusers` is set.

If the value has grown between runs of the same script, it probably means that VUs are taking longer to complete their scenarios, which would usually be caused by higher response times from the server.

**Reply by metallicalfa2** [reply]
Thanks.
